docs/schemaform/form-config.md

```diff
@@ -16,7 +16,8 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
   // The prefix for Google Analytics events that are sent for different form actions
   trackingPrefix: '',
 
-  // The title of the form. Displayed on all pages
+  // The title of the form. Displayed on all pages.
```

Comment:
Is this true? I thought `onReview` was only for chapter titles. Maybe you meant to put this there instead of on the form title documentation?